docs/mllib-evaluation-metrics.md

```diff
@@ -439,21 +439,21 @@ $$rel_D(r) = \begin{cases}1 & \text{if $r \in D$}, \\ 0 & \text{otherwise}.\end{
         Precision at k
       </td>
       <td>
-        $p(k)=\frac{1}{M} \sum_{i=0}^{M-1} {\frac{1}{k} \sum_{j=0}^{\text{min}(\left|D\right|, k) - 1} rel_{D_i}(R_i(j))}$
+        $p(k)=\frac{1}{M} \sum_{i=0}^{M-1} {\frac{1}{k} \sum_{j=0}^{\text{min}(\left|R_i\right|, k) - 1} rel_{D_i}(R_i(j))}$
```

Comment:
Although we usually have k <= |D| and k <= |R_i|, I agree that this is technically what must have been meant. How about using Q here instead of |R_i|? it was already used in the following formula. You can change both, sure, either way, as long as they're consistent.

Reply:
Actually the Q will vary with i, that is why I gave the notation |R_i|.

Reply:
Thanks. updated.